ui/src/main/webapp/src/app/executions/execution-detail.component.html

```diff
@@ -47,6 +47,10 @@ <h3 i18n="Analysis status|Section">Status</h3>
                     <dt i18n="Analysis last modification">Last modification</dt>
                     <dd>{{execution.lastModified ? (execution.lastModified | date: 'short') : 'n/a'}}</dd>
                 </ng-container>
+                <ng-container *ngIf="execution.state == 'COMPLETED' && execution?.analysisContext?.generateStaticReports">
+                    <dt i18n="Analysis static report">Report</dt>
+                    <dd><a class="link" target="_blank" href="{{formatStaticReportUrl(execution)}}">Static Report <i class="fa fa-external-link"></i></a></dd>
```

Comment:
Yes, sure. I just used "Static report" because i'm used to call it this way.
You suggest to only have a link "Open report" in the first column instead of the "Report" label or let the link be in the second column only changing the text to "Open report"?